roles/ezproxy/tasks/main.yml

```diff
@@ -174,47 +142,53 @@
   when:
     - running_on_server
 
-- name: Ezproxy | install ezproxy startup script (requires root)
-  ansible.builtin.command: /var/local/ezproxy/ezproxy -si
-  become: true
-  become_user: 'root'
-  changed_when: false
-  when: running_on_server
-
-
 - name: Ezproxy | add shib config user
   ansible.builtin.template:
     src: "shibuser.txt.j2"
     dest: /var/local/ezproxy/shibuser.txt
-    mode: 0644
+    mode: "0644"
     owner: root
     group: root
 
+- name: Ezproxy | add shib config user
+  ansible.builtin.template:
+    src: "princeton_allow.txt.j2"
+    dest: /var/local/ezproxy/princeton_allow.txt
+    mode: "0644"
+    owner: root
+    group: root
 
 - name: Ezproxy | copy a systemd service
   ansible.builtin.template:
     src: "ezproxy.service.j2"
-    dest: /var/local/ezproxy/ezproxy.service
+    dest: /lib/systemd/system/ezproxy.service
     owner: root
     group: root
     mode: a+x
   changed_when: false
   when:
     - running_on_server
+
+- name: Ezproxy | reload systemd to recognize new service
+  ansible.builtin.systemd:
+    daemon_reload: true
+  when:
+    - running_on_server
   notify: reload ezproxy
```

Comment:
In this task we are reloading systemd as a whole and if this task returns `changed` we will run a handler to reload ezproxy specifically . . . but a task or two below this one, we are also restarting ezproxy. Do we still need to notify the reload task here?